scholarx/2022/index.html

```diff
@@ -210,12 +210,19 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentors</h1>
                                         data-target="#collapseTwo"
                                         aria-expanded="false"
                                         aria-controls="collapseTwo">
-                                    <div class="question-heading">
-                                        What are the eligibility requirements?
-                                    </div>
-                                    <div class="text-primary collapse-sign">
-                                        <i class="plusMinusSign fa fa-plus float-right"></i>
-                                    </div>
+                                        <table class="faq-table">
+                                            <tr>
+                                                <td>
+                                                    <div class="question-heading">
+                                                        What are the eligibility 
+                                                    </div>requirements?
```

Comment:
Ok then for now put this inside the div tag and separate them with a `<br>` tag @janithlahirukariyawasam